wallet/src/ui/styles/global.scss

```diff
@@ -1,4 +1,5 @@
 @import url('bootstrap-icons');
+@import url('https://fonts.googleapis.com/css2?family=Space+Mono:wght@400;700&display=swap');
```

Comment:
I think we could remove the custom font at the moment, since we expect to have new designs. If you want to use it please change it to include the font from a local resource